scripts/cybershake/auto_submit.py

```diff
@@ -135,179 +229,109 @@ def submit_task(
             srf_name=run_name,
-            mgmt_db_loc=mgmt_db_location,
+            mgmt_db_loc=root_folder,
             script_location=os.path.expandvars("$gmsim/workflow/scripts/clean_up.sl"),
             output_file=os.path.join(sim_dir, "clean_up.out"),
             error_file=os.path.join(sim_dir, "clean_up.err"),
         )
         shared.submit_sl_script(
             script,
-            const.ProcessType.clean_up.str_value,
-            const.State.queued.str_value,
-            mgmt_db_location,
+            const.ProcessType.clean_up.value,
+            sim_struct.get_mgmt_db_queue(root_folder),
             run_name,
-            submitted_time,
             submit_yes=True,
             target_machine=const.HPC.mahuika.value,
         )
 
 
-# TODO: Requires updating, currently not working
-# def check_params_uncertain(params_uncertain_path):
-#     if not os.path.isfile(params_uncertain_path):
-#         print(params_uncertain_path, " missing, creating")
-#         cmd = "python $gmsim/workflow/scripts/submit_emod3d.py --set_params_only"
-#         call(cmd, shell=True)
-#         print(cmd)
-
-
-def get_vmname(srf_name):
-    """
-        this function is mainly used for cybershake perpose
-        get vm name from srf
-        can be removed if mgmt_DB is updated to store vm name
-    """
-    vm_name = srf_name.split("_")[0]
-    return vm_name
-
-
-def main():
-    parser = argparse.ArgumentParser()
-    parser.add_argument("run_folder", type=str, help="folder to the collection of runs")
-    parser.add_argument(
-        "--n_runs",
-        default=None,
-        type=list,
-        nargs="+",
-        help="The number of processes each machine can run at once. If a single value is given this is used for all "
-        "machines, otherwise one value per machine must be given. The current order is: {}".format(
-            (x.str_value for x in const.HPC)
-        ),
-    )
-
-    # cybershake-like simulations store mgmnt_db at different locations
-    parser.add_argument("--single_sim", nargs="?", type=str, const=True)
-    parser.add_argument(
-        "--config",
-        type=str,
-        default=None,
-        help="a path to a config file that constains all the required values.",
-    )
-    parser.add_argument("--user", type=str, default=None)
-    parser.add_argument("--no_im", action="store_true")
-    parser.add_argument("--no_merge_ts", action="store_true")
-    parser.add_argument("--no_clean_up", action="store_true")
-
-    args = parser.parse_args()
-
-    mgmt_db_location = args.run_folder
-    db = connect_db(mgmt_db_location)
+def main(args):
+    root_folder = args.root_folder
+    queue_folder = sim_struct.get_mgmt_db_queue(root_folder)
+    mgmt_db = MgmtDB(sim_struct.get_mgmt_db(root_folder))
 
     # Default values
-    oneD_mod, hf_vs30_ref, binary_mode, hf_seed = default_1d_mod, None, True, None
+    oneD_mod, hf_vs30_ref, binary_mode, hf_seed = DEFAULT_1D_MOD, None, True, None
```

Comment:
oneD_mod, hf_vs30ref can be removed, since they are not used in anywhere and cannot be override. 
(simulation reads the value from params.yaml)